core/src/test/java/org/elasticsearch/search/fetch/subphase/highlight/HighlighterSearchIT.java

```diff
@@ -1358,14 +1359,13 @@ public void testCommonTermsTermVector() throws IOException {
     public void testPhrasePrefix() throws IOException {
         Builder builder = Settings.builder()
                 .put(indexSettings())
-                .put("index.mapping.single_type", false)
                 .put("index.analysis.analyzer.synonym.tokenizer", "whitespace")
                 .putArray("index.analysis.analyzer.synonym.filter", "synonym", "lowercase")
                 .put("index.analysis.filter.synonym.type", "synonym")
                 .putArray("index.analysis.filter.synonym.synonyms", "quick => fast");
 
-        assertAcked(prepareCreate("test").setSettings(builder.build()).addMapping("type1", type1TermVectorMapping())
-                .addMapping("type2",
+        assertAcked(prepareCreate("test").setSettings(builder.build()).addMapping("type1", type1TermVectorMapping()));
```

Comment:
Can you rename these indices? I know I'll get `test` and `test1` mixed up. Even `test1` and `test2` would be better.